Automattic/wc-calypso-bridge
# Add simple front-end tooling

We've had [a recent issue](https://github.com/Automattic/wc-calypso-bridge/issues/640) that requires the use of front-end tooling to solve properly, and it seems that this need will only grow in the future. This PR adds simple dev tooling using the architecture provided by Gutenberg and `npm run create-wc-extension`

I've also amended the README to include instructions to utilize this tooling.

**Test Instructions**
- Checkout branch
- Run `npm install` and `npm start`
- Uncomment `console.info` statement in `index.js` file
- Confirm that script is loaded and log statement is printed
- Check live reloading by changing log statement and reloading
